shell/platform/windows/client_wrapper/flutter_engine_unittests.cc

```diff
@@ -121,4 +129,21 @@ TEST(FlutterEngineTest, GetMessenger) {
   EXPECT_NE(engine.messenger(), nullptr);
 }
 
+TEST(FlutterEngineTest, DartEntrypointArgs) {
+  DartProject project(L"data");
+  std::vector<std::string> arguments(2);
+  arguments[0] = "one";
+  arguments[1] = "two";
+  project.set_dart_entrypoint_arguments(arguments);
+  testing::ScopedStubFlutterWindowsApi scoped_api_stub(
+      std::make_unique<TestFlutterWindowsApi>());
+  auto test_api = static_cast<TestFlutterWindowsApi*>(scoped_api_stub.stub());
```

Comment:
Generally I've been putting the stub boilerplate at the very beginning so it's easy to skip past to look for the test-specific code.